internal/gojs/compiler_test.go

```diff
@@ -96,6 +102,19 @@ func TestMain(m *testing.M) {
 		log.Panicln(err)
 	}
 
+	// In order to avoid race condition on scheduleTimeoutEvent, we need to set the memory max
+	// and WithMemoryCapacityFromMax(true) above.
+	// https://github.com/tetratelabs/wazero/issues/992
+	//
+	// TODO: Maybe add WithMemoryMax API?
```

Comment:
We had something like this once, but I wonder if it would interfere with caching? Changing the max would be slightly different than the source wasm, where capacity is undefined in wasm anyway.